Resources/services/subscribers.xml

```diff
@@ -42,18 +32,23 @@
             <tag name="shopware.event_subscriber"/>
             <argument type="service" id="adyen_payment.components.configuration"/>
             <argument type="service" id="adyen_payment.components.adyen.payment.method"/>
-            <argument type="service" id="adyen_payment.components.payment_method_service"/>
-            <argument type="service" id="session"/>
-            <argument type="service" id="models"/>
-            <argument type="service" id="snippets"/>
-            <argument type="service" id="front" />
-            <argument type="service" id="adyen_payment.components.manager.adyen_manager" />
             <argument type="service" id="adyen_payment.components.data_conversion" />
+            <argument type="service" id="AdyenPayment\Components\Adyen\PaymentMethod\EnrichedPaymentMeanProvider"/>
+            <argument type="service" id="AdyenPayment\Components\Adyen\Builder\PaymentMethodOptionsBuilder"/>
+        </service>
+        <service id="adyen_payment.subscriber.checkout.add_google_pay_config_to_view_subscriber" class="AdyenPayment\Subscriber\Checkout\AddGooglePayConfigToViewSubscriber">
+            <tag name="shopware.event_subscriber"/>
+            <argument type="service" id="adyen_payment.components.configuration"/>
         </service>
-        <service id="adyen_payment.subscriber.frontend_payment_name_subscriber" class="AdyenPayment\Subscriber\FrontendPaymentNameSubscriber">
+        <service id="AdyenPayment\Subscriber\Checkout\RegisterConfirmSnippetsSubcriber"
+                 class="AdyenPayment\Subscriber\Checkout\RegisterConfirmSnippetsSubcriber">
             <tag name="shopware.event_subscriber"/>
-            <argument type="service" id="adyen_payment.components.payment_method_service"/>
-            <argument type="service" id="adyen_payment.logger"/>
+            <argument type="service" id="snippets"/>
+        </service>
+        <service id="AdyenPayment\Subscriber\Checkout\RegisterPaymentSnippetsSubcriber"
+                 class="AdyenPayment\Subscriber\Checkout\RegisterPaymentSnippetsSubcriber">
+            <tag name="shopware.event_subscriber"/>
+            <argument type="service" id="snippets"/>
         </service>
```

Comment:
This service exists under
config file: AdyenPayment/Resources/services/subscribers.xml

```xml
<service id="adyen_payment.subscriber.checkout.add_google_pay_config_to_view_subscriber" class="AdyenPayment\Subscriber\Checkout\AddGooglePayConfigToViewSubscriber">
  <tag name="shopware.event_subscriber"/>
  <argument type="service" id="adyen_payment.components.configuration"/>
</service>
```

Reply:
@bortefi thanks, I missed that, maybe because it is being declared differently with a custom name but when I tested it seemed it was not being loaded, please check

Reply:
Do we need to address this before merging @bortefi @DaveBueds?

Reply:
Seemed to work for me now :)